src/components/Typography/Text.js

```diff
@@ -0,0 +1,168 @@
+import React from 'react'
+import Radium from 'radium'
+import PropTypes from 'prop-types'
+import { colors } from '../../styles'
+
+// Not using `margin` or `padding` shorthand  to avoid any future
+// Radium warnings with combining shorthand/longhand properties.
+const baseStyles = {
+  paddingTop: 0,
+  paddingRight: 0,
+  paddingBottom: 0,
+  paddingLeft: 0,
+  marginTop: 0,
+  marginRight: 0,
+  marginBottom: 0,
+  marginLeft: 0,
+  color: colors.GRAY_13
+}
+
+const variantStyles = {
+  'T.92': {
+    fontSize: '92px',
+    lineHeight: '138px',
+    letterSpacing: '0',
+    fontWeight: 400
+  },
+  'T.82': {
+    fontSize: '82px',
+    lineHeight: '123px',
+    letterSpacing: '0',
+    fontWeight: 400
+  },
+  'T.72': {
+    fontSize: '72px',
+    lineHeight: '108px',
+    letterSpacing: '0',
+    fontWeight: 400
+  },
+  'T.64': {
+    fontSize: '64px',
+    lineHeight: '96px',
+    letterSpacing: '0',
+    fontWeight: 400
+  },
+  'T.58': {
+    fontSize: '58px',
+    lineHeight: '87px',
+    letterSpacing: '0',
+    fontWeight: 400
+  },
+  'T.46': {
+    fontSize: '46px',
+    lineHeight: '69px',
+    letterSpacing: '0',
+    fontWeight: 400
+  },
+  'T.36': {
+    fontSize: '36px',
+    lineHeight: '54px',
+    letterSpacing: '0.03em',
+    fontWeight: 400
+  },
+  'T.28': {
+    fontSize: '28px',
+    lineHeight: '42px',
+    letterSpacing: '0.05em',
+    fontWeight: 700
+  },
+  'T.22': {
+    fontSize: '22px',
+    lineHeight: '32px',
+    letterSpacing: '0.03em',
+    fontWeight: 400
+  },
+  'T.18': {
+    fontSize: '18px',
+    lineHeight: '32px',
+    letterSpacing: '0.03em',
+    fontWeight: 400
+  },
+  'T.16': {
+    fontSize: '16px',
+    lineHeight: '24px',
+    letterSpacing: '0.03em',
+    fontWeight: 400
+  },
+  'T.14': {
+    fontSize: '14px',
+    lineHeight: '21px',
+    letterSpacing: '0.03em',
+    fontWeight: 400
+  },
+  'T.12': {
+    fontSize: '12px',
+    lineHeight: '18px',
+    letterSpacing: '0.05em',
+    fontWeight: 400
+  },
+  'T.11': {
+    fontSize: '11px',
+    lineHeight: '16px',
+    letterSpacing: '0.05em',
+    fontWeight: 400
+  }
+}
+
+const htmlTagMapping = {
+  'T.92': 'h1',
+  'T.82': 'h1',
+  'T.72': 'h1',
+  'T.64': 'h1',
+  'T.58': 'h1',
+  'T.46': 'h1',
+  'T.36': 'h2',
+  'T.28': 'h2',
+  'T.22': 'h2',
+  'T.18': 'p',
+  'T.16': 'p',
+  'T.14': 'p',
+  'T.12': 'p',
+  'T.11': 'p'
+}
+
+// Matches the Open Sans font weights
+// https://fonts.google.com/specimen/Open+Sans?selection.family=Open+Sans
+const fontWeights = {
```

Comment:
should we be using these const in the font definitions ? It feels like the ones defined in font are just defaults out of this list 🤷‍♀️